README.md

```diff
@@ -1,13 +1,39 @@
-# Terradactile
+<br />
+<p align="center">
+  <a href="https://github.com/sparkgeo/template-barebones">
+    <img src="./images/sparkgeo-logo.png" width="100">
+  </a>
+  <h3 align="center">Terradactile Lambda</h3>
+</p>
+
+
+<!-- TABLE OF CONTENTS -->
+### Table of Contents
+
+* [About this Program](#about-the-project)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation and Deployment](#installation-and-deployment)
+* [Usage](#usage)
+* [Contact](#contact)
 
-Terradactile is an API and Lambda function for fetching, assembling, and processing elevation data.
 
-This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
+<!-- ABOUT THE PROJECT -->
+# About This Program
 
+Terradactile is an API and Lambda function for fetching, assembling, and processing elevation data.
+
+```markdown
+File Contents
 - terradactile/terradactile - Code for the application's Lambda function.
-- terradactile/template.yaml - A template that defines the application's AWS resources. **Update the environment variables for BUCKET and ALLOWED_ORIGINS.**
+- terradactile/template.yaml - A template that defines the application's AWS resources. 
```

Comment:
This code block is a bit weird to me. Consider not putting File Contents in a code block and leave it as text. Then use ` to identify that `terradactile/terradactile` and `terradactile/template.yaml` are paths.